homeassistant/components/homekit/__init__.py

```diff
@@ -168,8 +169,16 @@ def get_accessory(hass, driver, state, aid, config):
     if a_type is None:
         return None
 
+    message = "Add accessory {}".format(a_type)
+
     _LOGGER.debug('Add "%s" as "%s"', state.entity_id, a_type)
-    return TYPES[a_type](hass, driver, name, state.entity_id, aid, config)
+
+    accessory = TYPES[a_type](hass, driver, name, state.entity_id, aid, config)
+    accessory.sent_logbook_message(
+        EVENT_HOMEKIT_CHANGED, message
```

Comment:
I added this in there as I had a few times when I noticed something wasn't in Homekit I was expecting to be there. Normally speaking one would not run with debug and I had noticed that automations put in there an initial statement as well that they are now "active". 
This then allows an end-user to more easily see/conform that from HASS it is added as an accessory to HomeKit without having to go into log files and having to turn debug on first.

I personally like to be able to easily see in the logbook that the accessories have been added, and it is only once they go in there upon start hence it would not be spamming the logbook as such I think.

Let me know if you want to keep it there (my preference) or not, if not I'll just revert that piece of code. :-)

Reply:
Would you be ok with leaving it out for now and come to it later, once this is merged?
I would like to focus this PR one the `changed` messages, since this way it will already be quite large.

Reply:
Yep, removed that part.